components/proxy-service/internal/proxy/cache.go

```diff
@@ -0,0 +1,49 @@
+package proxy
+
+import (
+	gocache "github.com/patrickmn/go-cache"
+	"net/http/httputil"
+	"time"
+	"github.com/kyma-project/kyma/components/proxy-service/internal/authorization"
+)
+
+const cleanupInterval = 60
```

Comment:
Isn't that time a bit too much? Cleaning up cache more often would reduce amount of retries that will be made due to deprecated tokens.
What is your opinion on that?